docs/network/evm-nodes.md

```diff
@@ -70,35 +91,54 @@ You can initialize the node from a snapshot, initialize it from an existing Ethe
 
 ### From a snapshot
 
-To automatically download and import a snapshot, start the node with the `--init-from-snapshot` switch and the network to use, as in this example:
+You can import a snapshot into the EVM node in any of these ways:
 
-```bash
-octez-evm-node run observer \
-  --data-dir <EVM_DATA_DIR> \
-  --network testnet \
-  --init-from-snapshot
-```
+- Manually download and import the snapshot
+- Provide the URL of the snapshot to the node
+- Allow the node to download the snapshot automatically
 
-The node can take time to download and import the snapshot.
+You must download the appropriate snapshot for the network and mode.
+Rolling and full snapshots include 1 day of complete data.
```

Comment:
It’s a choice of our snapshot service, but you can create snapshots with more days. It’s just we currently run `full:1` and `rolling:1` nodes.